chart/k8gb/Chart.yaml

```diff
@@ -5,6 +5,7 @@ icon: https://www.k8gb.io/assets/images/icon-192x192.png
 type: application
 version: v0.8.8
 appVersion: v0.8.8
+kubeVersion: ">= 1.19.0-0"
```

Comment:
Maybe I didn't understand properly. In PR description you mentioned _Since k8s@1.22 the Ingress with version extensions/v1beta1 nor networking.k8s.io/v1beta1 are no longer supported_ . Is `>= 1.19.0-0` expected in Helm chart? shouldn't be `>= 1.22.0-0` ?

Reply:
`Ingress@v1` was introduced in `1.19` and `v1Beta1` was deprecated, since `1.22` `v1beta1` is gone. There was an overlap. Check this [compatibility matrix](https://github.com/kubernetes-sigs/external-dns#kubernetes-version-compatibility), this pretty much holds also for us.

In our case, it's gonna be:

| k8gb                    |      <= 0.8.x      |     >= 0.9.0 (or 1.0.0)      |
| ------------------------------ | :----------------: | :----------------: |
| Kubernetes <= 1.18             | :white_check_mark: |        :x:         |
| Kubernetes >= 1.19 and <= 1.21 | :white_check_mark: | :white_check_mark: |
| Kubernetes >= 1.22             |        :x:         | :white_check_mark: |

Reply:
thx for clarification @jkremser !